rules/no-guard/no-guard.md

```diff
@@ -0,0 +1,32 @@
+# effector/no-guard
+
+Any `guard` call could be replaced with `sample` call.
+
+```ts
+// 👎 could be replaced
+guard({ clock: trigger, soruce: $data, filter: Boolean, target: reaction });
+
+// 👍 makes sense
+sample({ clock: trigger, source: $data, filter: Boolean, target: reaction });
+```
+
+Nice bonus: `sample` is extendable. You can add transformation by `fn`.
+
+```ts
+// 👎 could be replaced
+guard({
+  clock: trigger,
+  soruce: $data.map((data) => data.length),
+  filter: Boolean,
+  target: reaction,
+});
+
+// 👍 makes sense
+sample({
+  clock: trigger,
+  soruce: $data,
```

Comment:
same

Reply:
Yeah, but I have no access) 

// Please make sure you have the correct access rights

Reply:
You have to fork repo, commit changes to your fork and send a PR.

Reply:
thanks) will do it now

Reply:
Done, waiting for your approve